config/config.yaml.dist

```diff
@@ -395,6 +395,8 @@ dex:
 #        exposeAdminKubeconfig: true
 #        ssh:
 #            generate: true
+#        # Enable create & update of EKS addons like coredns
+#        enableAddons: true
```

Comment:
**Recommended**: IMO the documented value **SHOULD** reflect the default value used without the configuration being present, which I suppose is `false` IIRC.
```suggestion
#        enableAddons: false
```